features/semantic-versioning.feature

```diff
@@ -0,0 +1,41 @@
+Feature: Calculating a version number based on semantic versioning
+
+  Scenario: Releasing a major version
+    Given a release on this branch with version "1.0.0"
```

Comment:
I had to read this line a few times to understand

* `Given the latest release on this branch is x.x.x`
* `Given the latest tag on this branch is x.x.x`